refactor(linter): move imports

overlookmotel opened the pull request:
Pure refactor. `run_external_rules` used to be feature-gated, so had a bunch of `use` statements in its body. But it's not feature-gated any more, so move them up to top level.

Also move a few imports in tests.

camc314 commented:
### Merge activity

* **Sep 24, 2:49 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

overlookmotel committed:
refactor(linter): move imports (#14085)

Pure refactor. `run_external_rules` used to be feature-gated, so had a bunch of `use` statements in its body. But it's not feature-gated any more, so move them up to top level.

Also move a few imports in tests.
